### DISCUSSION TITLE
DTM v2.6 issue

### DISCUSSION THREAD

**Discussion by atlaschen1113** [olleheugene/nRF-DTM#6] [General]
Hi
Download this version last month and it cannot work with DK.
Could you check it?
Thanks.

**Comment by olleheugene** [comment]
Hi,

Thanks for your feedback!
I found that x64 has problem to run the DTM.
Would you please run with x86 version or previous version while I fix the issue?

Thanks.

**Comment by olleheugene** [comment]
Would you please try with [v2.6.1](https://github.com/olleheugene/nRF-DTM/raw/master/Release/Windows_x64/nRF_DTM_x64.exe)?

**Reply by atlaschen1113** [reply]
Hi 
It's working in v2.6.1.
Thanks for updating.
